app/views/albums/index.html.erb

```diff
@@ -0,0 +1,18 @@
+
+<table class = "table table-hover">
+  <tr>
+    <th> Ranking </th>
+    <th> Name </th>
+    <th> Upvote </th>
+  </tr>
+
+  <% @albums.each do |album| %>
+  <tr>
+    <td><%= album.rank %></td>
+    <td><%= link_to album.name, album_path(album.id)%></td>
```

Comment:
Passing ```album.id``` into the ```album_path``` works, but you can also pass in just the album itself and the Rails routing magic will resolve it correctly. See more on passing objects into routes [here] (http://guides.rubyonrails.org/routing.html#creating-paths-and-urls-from-objects).